docs2/site/docs/getting-started/dependency-injection.md

```diff
@@ -198,79 +283,3 @@ public class StarWarsQuery : ObjectGraphType
-
-## Using UserContext Scoped IServiceProvider
-
-**Be Aware**: Exposing the service locator is considered an Anti-Pattern. See [Service Locator is an Anti-Pattern](https://blog.ploeh.dk/2010/02/03/ServiceLocatorisanAnti-Pattern/)
-
-You can add the relevant scoped `IServiceProvider` when creating the `UserContext` in order to run `IDocumentExecuter`.
-
-E.g. In ASP.NET Core [GraphQL.Harness](https://github.com/graphql-dotnet/graphql-dotnet/tree/master/src/GraphQL.Harness) example, you could add `IServiceProvider` to the [GraphQLUserContext](https://github.com/graphql-dotnet/graphql-dotnet/blob/master/src/GraphQL.Harness/GraphQLUserContext.cs) class, and then set it to the `HttpContext.RequestServices` when configuring the `GraphQLSettings.BuildUserContext`
-
-
-1. Add ServiceProvider to the UserContext
-
-    ```csharp
-    public class GraphQLUserContext: Dictionary<string, object>
-    {
-        ...
-        IServiceProvider ServiceProvider { get; set; }
-        ...
-    }
-    ```
-
-2. Set it to the `HttpContext.RequestServices` when building the `UserContext`.
-
-    > N.B The below uses [GraphQLSettings](https://github.com/graphql-dotnet/graphql-dotnet/blob/master/src/GraphQL.Harness/GraphQLSettings.cs) from the [GraphQL.Harness](https://github.com/graphql-dotnet/graphql-dotnet/tree/master/src/GraphQL.Harness) example.
-
-    ```csharp
-    public void ConfigureServices(IServiceCollection services)
-    {
-        services.Configure<GraphQLSettings>(settings => settings.BuildUserContext = ctx => new GraphQLUserContext
-        {
-            User = ctx.User,
-            ServiceProvider = ctx.RequestServices
-        });
-    }
-    ```
-
-3. Use it in resolvers
-
-    ```csharp
-    public class StarWarsQuery : ObjectGraphType<object>
-    {
-        public StarWarsQuery()
-        {
-            ...
-            Field<CharacterInterface>("hero",
-                resolve: context =>
-                    ((GraphQLUserContext)context.UserContext)
-                    .GetRequiredService<IDroidRepo>()
-                    .GetDroidByIdAsync("3")
-            );
-
-            Field<CharacterInterface>("hero", resolve: context => context);
-        }
-    }
-    ```
-
- You could always create an extension method:
-
- ```csharp
-public static class GraphQLExtensions
-{
-    public static T GetRequiredService<T>(this IResolveFieldContext context) =>
-        ((GraphQLUserContext)context.UserContext).ServiceProvider.GetRequiredService<T>();
-}
- ```
-
-and use it as such:
-
-```csharp
-Field<CharacterInterface>("hero",
-            resolve: context =>
-                context
-                .GetRequiredService<IDroidRepo>()
-                .GetDroidByIdAsync("3")
-        );
-```
```

Comment:
It's probably good that we won't talk anymore about using `UserContext` for working with scopes.